Applies the fix to ykval-db-pdo.php which fixes the ykval-queue init script
so that it does not fail if a database is not initially available.
This fix is in upstream, in the unreleased 1.12 version.
Index: yubikey-val-dpkg/ykval-db-pdo.php
===================================================================
--- yubikey-val-dpkg.orig/ykval-db-pdo.php	2013-01-31 16:52:01.000000000 +0100
+++ yubikey-val-dpkg/ykval-db-pdo.php	2013-01-31 16:52:17.000000000 +0100
@@ -28,6 +28,7 @@
     $this->db_username=$db_username;
     $this->db_password=$db_password;
     $this->db_options=$db_options;
+    $this->result = null;
 
     $this->myLog=new Log($name);
   }
@@ -96,7 +97,7 @@
    */
   public function closeCursor($result=null){
     if(!$result) $result = $this->result;
-    $result->closeCursor();
+    if($result) $result->closeCursor();
   }
 
   public function truncateTable($name)
